server/config.ts

```diff
@@ -74,6 +74,14 @@ export default {
       },
       agent: new AgentConfig(Number(get('PRISON_API_TIMEOUT_DEADLINE', 20000))),
     },
+    prisonerSearchApi: {
+      url: get('PRISONER_SEARCH_API_URL', 'http://localhost:8082', requiredInProduction),
+      timeout: {
+        response: Number(get('OFFENDER_SEARCH_API_TIMEOUT_SECONDS', 20000)),
+        deadline: Number(get('OFFENDER_SEARCH_API_TIMEOUT_SECONDS', 20000)),
+      },
+      agent: new AgentConfig(Number(get('OFFENDER_SEARCH_API_TIMEOUT_DEADLINE', 20000))),
```

Comment:
you probably need to change these env var names as well as PRISONER_SEARCH_API_URL ?